### DISCUSSION TITLE
Error: unable to find CUDA device 0 among 2 EGL devices in total when testing Habitat-Lab on WSL 2

### DISCUSSION THREAD

**Discussion by ThanhIT0410** [facebookresearch/habitat-lab#1698] [Q&A]
Hello, I am trying to install habitat-lab for some research. I have completed all the install steps in the installation section, but when it comes to the testing part, when I run the command

> python examples/interactive_play.py --never-end

I am getting following errors:

> [15:21:55:118119]:[Warning]:[Metadata] SceneDatasetAttributesManager.cpp(124)::loadAndValidateMap : `navmesh_instances` 
Value : `navmeshes/v3_sc4_staging_20.navmesh` not found on disk as absolute path or relative to `data/replica_cad`
Platform::WindowlessEglApplication::tryCreateContext(): unable to find CUDA device 0 among 2 EGL devices in total
WindowlessContext: Unable to create windowless context

Please let me know how to resolve this. Thank you!

**Comment by Ethan2415** [comment]
I get the same problem

**Reply by dantes-ai** [reply]
Facing similar issue on WSL2, Win11. Has anyone found a solution?

**Reply by xxR627** [reply]
我在尝试使用docker，但不知道能不能解决
